UI/Hotkeys.cpp

```diff
@@ -233,6 +239,9 @@ void Hotkey::ReadFromOptions(OptionsDB& db) {
 
         hotkey.m_key = key_modkey_pair.first;
         hotkey.m_mod_keys = key_modkey_pair.second;
+
+        DebugLogger()  <<__func__ << "Added hotkey '" << hotkey.m_key  << "' with modifiers '"
```

Comment:
I think it was concluded that __func__ was not portable, so if you really need it, write it out explicitly.